theories/Makefile

```diff
@@ -8,7 +8,7 @@ all install clean: Makefile.coq
 	+$(MAKE) -f $< $(ROCQMAKEOPTIONS) $@
 
 Makefile.coq: _CoqProject
-	$(ROCQMAKEFILE) $(ROCQMAKEFILEOPTIONS) -f $< $(shell find . -name "*.v") -o $@
+	$(ROCQMAKEFILE) $(ROCQMAKEFILEOPTIONS) -f $< $(shell find . -name "*.v" | grep -v "/[.][^/]*$$") -o $@
```

Comment:
I'd rather change the find command than postprocess with grep

Reply:
Why not